app/services/crypto_coin.js

```diff
@@ -7,3 +7,13 @@ exports.addCoin = (currency, userId) =>
     logger.error(err.message);
     return Promise.reject(errors.databaseError('Error with data base'));
   });
+
+exports.getCoinsByUser = id =>
+  CryptoCoin.findAll({
+    where: { userId: id },
```

Comment:
You can replace **id** for **userId**, id seems to refer to the model CryptoCoin